### DISCUSSION TITLE
Create interval timer

### DISCUSSION THREAD

**Discussion by ghost** [vydimitrov/react-countdown-circle-timer#90] [Q&A]
Hi,
I'm trying to implement an interval timer, where the duration changes.
Is there any way to do so, so once the timer is completed the duration changes?
Thanks a lot!
Abby

**Comment by vydimitrov** [comment]
Hey, here is your solution https://codesandbox.io/s/serene-moon-n0y6x?file=/src/index.js

**Reply by fio-la** [reply]
Is this possible for react native without using react.js?

**Reply by fio-la** [reply]
Actually I wanted to reset the timer while its running with help of a button. What it does in the file you refer is startig again after it completes, no? Or is the functionallity similar? Thanks

**Reply by vydimitrov** [reply]
The idea is the same, you just need to pass a new key prop. It works the same for react-native. Please read the docs.

**Reply by fio-la** [reply]
Thanks a lot
